aces/Loris
# [Electrophysiology Browser] New module

## Brief summary of changes

This PR includes SQL changes and code required for the new Electrophysiology Browser module. 

Screenshot of the menu filter page (reactified and data frameworked):
![Screen Shot 2019-09-17 at 5 20 43 PM](https://user-images.githubusercontent.com/1402456/65080730-f403d580-d96f-11e9-9403-ec49e7a65897.png)

Screenshot of the view session page:
![Screen Shot 2019-09-17 at 5 21 07 PM](https://user-images.githubusercontent.com/1402456/65080748-ffef9780-d96f-11e9-98bf-8e88b8b5fb2c.png)

### IMPORTANT

This has been a wonderful team work and I want the authors to be recognized as most of the work to create the module has been done by them :):
- @um4r12 for the original back end PHP code
- @maltheism for the react front end code
- @christinerogers, Derek for the UI and design
- @cmadjar, upgrade of the code to 21.0, to use the latest react for the menu filter, to use the data framework for the menu filter, few minor bug fixes, design, SQL etc...


#### Testing instructions (if applicable)

1. First, make sure to reload the latest RaisinBread that include EEG datasets (should be included in this branch since I rebased it after the RB dataset got merged)
2. Go to the Electrophysiology menu tab and select the Electrophysiology Browser to direct you to the new module. 
3. Have fun clicking and checking the pages :) 

#### Links to related tickets (GitHub, Redmine, ...)

* https://github.com/aces/Loris/issues/5031
